content/en/docs/reference/config/analysis/ist0108/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: UnknownAnnotation
+layout: analysis-message
+---
+
+This message occurs when you attach an annotation to a namespace, but Istio does not recognize the annotation.
```

Comment:
This sounds like its looking at all annotations, but I think this is only looking at annotations with `istio.io` as the domain for the annotations

Reply:
fixed